src/content/basics/nodepools/index.md

```diff
@@ -87,20 +87,36 @@ spec:
     giantswarm.io/machine-deployment: a1b2c
 ```
 
+A similar example for an Azure cluster:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+  nodeSelector:
+    giantswarm.io/machine-pool: a1b2c
+```
+
 You can assign workloads to node pools in a more indirect way too. This is achieved by using other node attributes which are
 specified via the node pool and which are exposed as node labels.
 
 For example: In a case where you have node pools with one instance type. Using a `nodeSelector` with the label `node.kubernetes.io/instance-type` (or in Kubernetes before v1.17: `beta.kubernetes.io/instance-type`) you can assign workloads to matching nodes only.
 
 Another example: In a case where you have different node pools using different availability zones. With a `nodeSelector` using the label `topology.kubernetes.io/zone` (or in Kubernetes before v1.17: `failure-domain.beta.kubernetes.io/zone`) you can assign your workload to the nodes in a particular availability zone.
 
+
 ## Node pool deletion
 
 You can delete a node pool at any time using the Giant Swarm API and user interfaces. When a node pool gets deleted the following things will happen:
 
 - nodes in the pool will be marked as unschedulable and then drained, resulting in Pods being unassigned from the nodes
```

Comment:
Does this mean the nodes are node drained before you remove them? That means the will have some downtime when a node pool is deleting right?

Reply:
@whites11 , I need help 😁

Reply:
yes @omegas27 we took the decision to not implement this feature, because we don't do it with downscaling anyway (azure lifecycle callbacks don't allow this).